build/goreleaser/linux/centos_7_amd64.yml

```diff
@@ -6,7 +6,7 @@
       - linux-ctl-amd64
       - linux-service-amd64
     package_name: newrelic-infra
```

Comment:
This package_name also needs to be parametrized to be fips or non-fips

Reply:
Heyo!  I added this in commit [e5ce704](https://github.com/newrelic/infrastructure-agent/pull/1942/commits/e5ce704acfbdc33e88333f7f148a7bf84d6fb3de)

Reply:
Running a build to make sure its still successful

Reply:
Build successfully ran